src/nvim/options.lua

```diff
@@ -831,10 +831,11 @@ return {
     },
     {
       full_name='foldcolumn', abbreviation='fdc',
-      type='number', scope={'window'},
+      type='string', scope={'window'},
```

Comment:
it pains me, but ....... this ... is going to break some plugin. Perhaps we can merge this on master and see if it breaks anything that actually matters.

Reply:
I know, even from a code point of view, having an other option "autofold" would have been easier to implement (instead of dealing with string allocation) but I wanted to remain consistent with the signcolumn syntax.

Reply:
Yeah, I'm glad you did it this way.